lib/data-generators/version-commit.js

```diff
@@ -21,18 +21,26 @@ module.exports = CoreObject.extend({
     }
 
     var info = gitRepoInfo(path);
-    var sha = info.sha.slice(0, 8);
+    var sha = (info.sha || '').slice(0, 8);
+    var log = this._plugin.log;
 
     return readFile(versionFile)
       .then(function(contents) {
         var json = JSON.parse(contents);
 
-        if (!json.version || !sha) {
+        if (!json.version) {
           return Promise.reject('Could not build revision with version `' + json.version + '` and commit hash `' + sha + '`');
         }
 
+        var versionString = json.version;
+        if (sha) {
+          versionString = versionString + '+' + sha;
+        } else {
+          log('Missing git commit sha, using package version as revisionKey');
```

Comment:
@cball could you add `{ color: 'yellow', verbose: true }` here?

example https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/blob/master/index.js#L33

@lukemelia this also reminds me that we can set log color for info and danger but not warning levels (it's actually not really a level yet though, so yellow should do for now)